source/ui/GuiApplication.ooc

```diff
@@ -0,0 +1,24 @@
+use ooc-base
+import DisplayWindow
+import EventLoop
+
+GuiApplication: class extends Application {
+	_window: DisplayWindow
```

Comment:
It's funny to have a `GuiApplication` with just one widget, but I guess we have to start somewhere...